libraries/exporters/src/main/java/org/eclipse/sw360/exporter/utils/TypeMappings.java

```diff
@@ -72,7 +72,7 @@ public String apply(License input) {
         return new Function<Todo, Integer>() {
             @Override
             public Integer apply(Todo input) {
-                return input.getTodoId();
+                return -1;
```

Comment:
What is this doing? Maybe add a comment, if this is needed. Or might there be another, better change that achieves the same?

Reply:
This is for mapping an instance (in this case a Todo) with an ID. Since we got rid of the id i made this return something in order for the code to compile. I would greatly appreciate a suggestion here.

Reply:
I was just wondering if we then need the mapping at all if the results cannot be distinguished anyway.

Or the Todos still have an ID fields, haven't they? Maybe use this?

Reply:
The id is a string, generated by CouchDB. My assumption is that since we do not have the todoId anymore, the mappings do not apply.

Reply:
Yeah, that might be correct. But wouldn't it be nice to remove the method then and change the calling code accordingly?
I did not check if this is possible or if it is called in some generic method.

Reply:
Removing or changing the calling code seems a lot of effort and probably some good understanding of the original design which i don't. Perhaps creating an issue mentioning this instead of fixing it in this PR would be more appropriate.